### ISSUE TITLE
[Safari] Toolbar icon isn't pixel-fitted on non-retina screens.

### ISSUE THREAD

**Issue by AllMelody** [uBlock-LLC/uBlock#1075]
## Steps to Reproduce

Install uBlock, add button to Safari toolbar.
## Symptoms

Icon looks nice on Retina screens, blurry on non-retina screens.

Compare, retina: 
![aaaa](https://cloud.githubusercontent.com/assets/869168/6772637/31da33f8-d0d9-11e4-9525-8cb0a115c2b7.png)
Non-retina magnified 2x:
![bbbb](https://cloud.githubusercontent.com/assets/869168/6772636/2b1b14f6-d0d9-11e4-9d4b-5224f50ccc19.png)

The U is way blurrier than the rest of the sharp lines in the other icons. This blog post explains what I mean by pixel-fitting: http://dcurt.is/pixel-fitting

```
uBlock version:
    0.9.2.0
Browser and version:
    Safari 8.0.4
OS and version:
    10.10.2
```

**Comment by chrisaljoudi** [comment]
@franzf fixed in 46b786d. Thanks.

**Comment by franzf** [comment]
I'm on 0.9.2.3 now, and the icon is still blurry. Seems like the cool new icon from the commit above didn't get applied to the actual button for some reason.

![screen shot 2015-03-29 at 21 30 14](https://cloud.githubusercontent.com/assets/869168/6889016/e2ec1ff4-d65a-11e4-9fd7-c27b83ae8259.png)
